apps/web/src/views/Farms/components/BCakeBoosterCard.tsx

```diff
@@ -11,8 +11,11 @@ import {
   Text,
   useTooltip,
   useMatchBreakpoints,
+  MessageText,
+  Message,
 } from '@pancakeswap/uikit'
 import useActiveWeb3React from 'hooks/useActiveWeb3React'
+import useBCakeProxyBalance from 'hooks/useBCakeProxyBalance'
```

Comment:
`useBCakeProxyBalance` should in the scope of Farm only. It should move to `Farms/hooks`